### ISSUE TITLE
Sourcepawn language server crashing

### ISSUE THREAD

**Issue by TheBv** [Sarrus1/sourcepawn-vscode#301]
# Basic informations

- **OS:** Win 11
- **VSCode version:** Latest
- **Extension version:** Latest

# Further Information

Sourcepawn language server just immediately crashes on startup. I feel like this wasn't an issue for me a couple months ago, but I might be wrong. 

# Error messages

```
[Info  - 6:48:58 PM] Connection to server got closed. Server will restart.
[2023-04-30T16:48:58.266Z INFO] Starting SourcePawn server version 0.9.10
Received config [Object {"disableSyntaxLinter": Bool(true), "includesDirectories": Array [String("C:\\Users\\Patrick\\sourcemodAPI\\addons\\sourcemod\\scripting\\include"), String("./includes")], "linterArguments": Array [], "mainPath": String(""), "spcompPath": String("C:\\Users\\Patrick\\sourcemodAPI\\addons\\sourcemod\\scripting\\spcomp.exe")}]
thread 'main' panicked at 'called `Result::unwrap()` on an `Err` value: ()', src\store.rs:149:61
stack backtrace:
note: Some details are omitted, run with `RUST_BACKTRACE=full` for a verbose backtrace.
[Error - 6:48:58 PM] The SourcePawn Language Server server crashed 5 times in the last 3 minutes. The server will not be restarted. See the output for more information.
```

**Comment by lsujk95** [comment]
I have the same problem:
`[Info  - 12:12:50 PM] Connection to server got closed. Server will restart.
[2023-05-12T10:12:50.393Z INFO] Starting SourcePawn server version 0.9.10
Received config [Object {"disableSyntaxLinter": Bool(true), "includesDirectories": Array [String("C:\\dev\\compilers\\sourcemod\\include"), String("scripting\\include")], "linterArguments": Array [], "mainPath": String(""), "spcompPath": String("C:\\dev\\compilers\\sourcemod\\spcomp.exe")}]
thread 'main' panicked at 'called `Result::unwrap()` on an `Err` value: ()', src\store.rs:149:61
stack backtrace:
note: Some details are omitted, run with `RUST_BACKTRACE=full` for a verbose backtrace.
thread '<unnamed>' panicked at 'called `Result::unwrap()` on an `Err` value: "SendError(..)"', C:\Users\runneradmin\.cargo\registry\src\index.crates.io-6f17d22bba15001f\lsp-server-0.6.0\src\stdio.rs:29:37
stack backtrace:
note: Some details are omitted, run with `RUST_BACKTRACE=full` for a verbose backtrace.
[Error - 12:12:50 PM] The SourcePawn Language Server server crashed 5 times in the last 3 minutes. The server will not be restarted. See the output for more information.`

**Comment by Sarrus1** [comment]
Fixed in [39f1bfc5b899bff12c7bbff1d380c17032aa326d](https://github.com/Sarrus1/sourcepawn-lsp/commit/39f1bfc5b899bff12c7bbff1d380c17032aa326d).
